04_MMU/error-codes.yaml

```diff
@@ -38,7 +38,7 @@ Errors:
 
 - code: "04103"
   title: "FSENSOR DIDNT TRIGGER"
-  text: "Filament sensor didn't trigger while loading the filament. Ensure the filament reached the fsensor and the sensor works."
+  text: "Filament sensor didn't trigger while loading the filament. Ensure the sensor is calibrated and the filament reached it."
```

Comment:
This is useful for the MK4 but on the MK3/S/+ may be confusing.
`P`age
`L`ine on page
``` 
P1|01234567890123456789
L1|Filament sensor
L2|didn'trigger while
L3|loading the
L4|filament. Ensure the

P2|01234567890123456789
L1|sensor is calibrated
L2|and the filament
L3|reached it.
```
Alternative
```
P1|01234567890123456789
L1|Filament sensor
L2|didn'trigger while
L3|loading the
L4|filament. Ensure the

P2|01234567890123456789
L1|filament reached it
L2|and works correctly.
```